### PR TITLE
Start preparation of Irish Tour

### PR DESCRIPTION
**Pull request by Claire-CConti**
### Description of Change

- Creating a documenation

### Test results

- N/A

### Who should know about the change?

/cc: @githubteacher

### CODE REVIEW

File: irish-tour.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Irish tour
+
+## Places to see
+
```
Comment (Claire-CConti) [inline irish-tour.md:4]: Connerama

**Review by Claire-CConti — COMMENTED** [review]
Sure

File: irish-tour.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Irish tour
```
Comment (mj580) [inline irish-tour.md:1]: Great idea!

File: irish-tour.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+## Places to see
```
Comment (mj580) [inline irish-tour.md:3]: very nice, indeed

**Review by mj580 — APPROVED** [review]
really nice

File: irish-tour.md (hunk 1 above)
Comment (hd2605) [reply irish-tour.md:4]: seems good

File: irish-tour.md (hunk 2 above)
Comment (MPMORRIER) [reply irish-tour.md:1]: trop tard

Comment (Claire-CConti) [reply irish-tour.md:1]: When is it better to go (weather conditions?)

File: irish-tour.md (hunk 3 above)
Comment (hd2605) [inline irish-tour.md:3]: seems ok

**Review by hd2605 — APPROVED** [review]

**Pull request merged**